components/server/src/workspace/workspace-starter.ts

```diff
@@ -1717,7 +1717,7 @@ export class WorkspaceStarter {
             "resource:" +
                 ScopedResourceGuard.marshalResourceScope({
                     kind: "envVar",
-                    subjectID: "*/*",
```

Comment:
Ah, nice to see it being such a small change :laughing: 

Should we here issue both scopes, to make sure we are backwards compatible? :thinking:

Reply:
@geropl we could issue both, but currently, we'd never use it. Maybe we can add it if/when we implement updating existing user-scopes variables (`*/*`) using some magic that lets you edit a variable scopes in either global approach - maybe something like "check existence of `VAR1` under `*/**`, if it doesn't exist, check `VAR1` under `*/*` and update accordingly".

Reply:
Ah right, because we are always "requesting" the `*/**` token here.

I just wonder how we can avoid additional turnarounds for existing customers... :thinking: Would you have time to sync?

Reply:
@geropl yup, does 12 work for ya?

Reply:
In the end, we decided to have a fallback when updating any variables scoped to `*/**` and not just error that the variable does not exist.

Reply:
Added backwards compatibility with `*/*` in [`a3b27b8` (#20279)](https://github.com/gitpod-io/gitpod/pull/20279/commits/a3b27b83a7c12f14d5ec4c6bc76a6a2901238ddc) @geropl. I also updated testing steps to reflect the changes